spring-cloud-starter-stream-sink-cassandra/src/main/java/org/springframework/cloud/stream/app/cassandra/util/NonValidatingSSLContextFactory.java

```diff
@@ -0,0 +1,32 @@
+package org.springframework.cloud.stream.app.cassandra.util;
+
+import javax.net.ssl.*;
+import java.security.GeneralSecurityException;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+
+public class NonValidatingSSLContextFactory {
```

Comment:
Some JavaDocs and your name as a author.

As well as in any other classes you have modified.